tensornetwork/network_operations.py

```diff
@@ -293,14 +293,27 @@ def split_node(
       max_truncation_err,
       relative=relative)
   sqrt_s = backend.sqrt(s)
-  u_s = u * sqrt_s
-  # We have to do this since we are doing element-wise multiplication against
-  # the first axis of vh. If we don't, it's possible one of the other axes of
-  # vh will be the same size as sqrt_s and would multiply across that axis
-  # instead, which is bad.
-  sqrt_s_broadcast_shape = backend.shape_concat(
-      [backend.shape_tensor(sqrt_s), [1] * (len(vh.shape) - 1)], axis=-1)
-  vh_s = vh * backend.reshape(sqrt_s, sqrt_s_broadcast_shape)
+  if backend.name != 'symmetric':
```

Comment:
We should NEVER have if statements based on the backed that don't raise errors. All of these difference should be handled by the backend classes. If they don't, we need to restructure the backends.

Reply:
Let me see how to fix this

Reply:
It looks like the only thing we nee to add is element wise multiplication.

Reply:
I'd like to avoid adding broadcasting support to `BlockSparseTensor` for now, because this could be a bit of a rabbit hole (I'm willing to add it later on though). The best alternative option I see for now is to add a `broadcast_left_multiply` and `broadcast_right_multiply` to the backends.  What do you think?

Reply:
fine by me.